endpoints/openrtb2/auction_test.go

```diff
@@ -103,7 +104,7 @@ func TestExplicitUserId(t *testing.T) {
 	// NewMetrics() will create a new go_metrics MetricsEngine, bypassing the need for a crafted configuration set to support it.
 	// As a side effect this gives us some coverage of the go_metrics piece of the metrics engine.
```

Comment:
The code paths in `getAccount()` are exercised by `TestRejectAccountRequired` as before, when the method was `validateAccount()`. It's what I used to ensure the existing behavior is maintained with the changes.

I can implement another set directed at `getAccount()` but it will be a copy of the cases from that other test, with the similar setup (because it needs endpoint deps).